admin_manual/ai/overview.rst

```diff
@@ -175,6 +175,80 @@ Apps can integrate their content with Context Chat to make it available for quer
 * *files*
 * `Analytics <https://apps.nextcloud.com/apps/analytics>`_
 
+.. _ai-overview_improve-ai-task-pickup-speed:
+
+Improve AI task pickup speed
+----------------------------
+
+Most AI tasks will be run as part of the background job system in Nextcloud which only runs jobs every 5 minutes by default.
+To pick up scheduled jobs faster you can set up background job workers that process AI tasks as soon as they are scheduled.
+
+Screen or tmux session
+^^^^^^^^^^^^^^^^^^^^^^
+
+Run the following occ command inside a screen or a tmux session, preferably 4 or more times for parallel processing of multiple requests by different or the same user (and as a requirement for some apps like context_chat).
+It would be best to run one command per screen session or per tmux pane to keep the logs visible and the worker easily restartable.
+
+.. code-block::
+
+   set -e; while true; do sudo -u www-data occ background-job:worker -v -t 60 "OC\TaskProcessing\SynchronousBackgroundJob"; done
+
+You may want to adjust the number of workers and the timeout to your needs. The above command will run with a timeout of 60 seconds which means any changes to the settings or the code will be picked up after 60 seconds (worst case scenario). This timeout does not, in any way, affect the processing or the timeout of the AI tasks.
```

Comment:
It's not obvious why the workers need to be restarted. Maybe one more sentence would make it clearer. Something like: `If the Php code or the Nextcloud settings values are changed while a worker is running, those changes won't be effective inside the runner. For that reason, the worker needs to be restarted regularly.`
Or even a bit more explicit and detailed. Wdyt?
This more detailed explanation would fit well in the intro (after "Most AI tasks will be run as part of the background job system in Nextcloud...").